src/main/java/seedu/address/model/group/Group.java

```diff
@@ -71,6 +71,19 @@ public void setPersons(Set<Name> editedPersonSet) {
         persons = editedPersonSet;
     }
 
+    /**
+     * Deletes the given person from this {@code group}.
```

Comment:
Write that after execution of this method, the `personNameToDelete` is guaranteed to be not in the group. Also add some note saying that no exceptions will be thrown.